mappings/net/minecraft/data/server/recipe/SmithingRecipeJsonBuilder.mapping

```diff
@@ -9,11 +9,11 @@ CLASS net/minecraft/class_5377 net/minecraft/data/server/recipe/SmithingRecipeJs
 		ARG 2 base
 		ARG 3 addition
 		ARG 4 result
-	METHOD method_29729 create (Lnet/minecraft/class_1856;Lnet/minecraft/class_1856;Lnet/minecraft/class_1792;)Lnet/minecraft/class_5377;
+	METHOD method_29729 createRecipe (Lnet/minecraft/class_1856;Lnet/minecraft/class_1856;Lnet/minecraft/class_1792;)Lnet/minecraft/class_5377;
 		ARG 0 base
 		ARG 1 addition
 		ARG 2 result
-	METHOD method_29730 criterion (Ljava/lang/String;Lnet/minecraft/class_184;)Lnet/minecraft/class_5377;
+	METHOD method_29730 withAddedCriterion (Ljava/lang/String;Lnet/minecraft/class_184;)Lnet/minecraft/class_5377;
```

Comment:
This modifies itself. So `criterion` is good enough.

Reply:
I will continue this particular discussion in #1938 to keep everything in one place.